resources/functionMetadata.php

```diff
@@ -1473,8 +1475,10 @@
 	'sqrt' => ['hasSideEffects' => false],
 	'stat' => ['hasSideEffects' => false],
 	'str_contains' => ['hasSideEffects' => false],
+	'str_decrement' => ['hasSideEffects' => false],
```

Comment:
Please see the top of this file:

https://github.com/phpstan/phpstan-src/blob/1bb4a34455dcca9239e7c6a50c36614daf718204/resources/functionMetadata.php#L3-L8